_ont/ont-zyxel-pmg3000-d20b.md

```diff
@@ -104,6 +104,19 @@ SW_VER1:0xabcedf
 ```
 {% include alert.html content="it's better to enter the software version in hex format, all lowercase precedeed by 0x." alert="Note"  icon="svg-info" color="blue" %}
 
+## See link speed
+This SFP have HSGMII enabled by default, link_status=5 HSGMII 2.5Gbit, link_status=5 SGMII 1Gbit:
```

Comment:
are you sure about this in lantiq stick should be `link_status=4` SGMII 1Gbit.

in any case you should put `link_status` inside inline code with ``

Reply:
sorry, I was wrong to write